apps/studio/pages/project/[ref]/settings/compute-and-disk.tsx

```diff
@@ -0,0 +1,28 @@
+import { DiskManagementForm } from 'components/interfaces/DiskManagement/DiskManagementForm'
+import SettingsLayout from 'components/layouts/ProjectSettingsLayout/SettingsLayout'
+import {
+  ScaffoldContainer,
+  ScaffoldDescription,
+  ScaffoldHeader,
+  ScaffoldTitle,
+} from 'components/layouts/Scaffold'
+import type { NextPageWithLayout } from 'types'
+
+const AuthSettings: NextPageWithLayout = () => {
+  return (
+    <>
+      <ScaffoldContainer>
+        <ScaffoldHeader>
+          <ScaffoldTitle>Compute and Disk</ScaffoldTitle>
+          <ScaffoldDescription>
+            Configure the compute and disk settings for your project.
+          </ScaffoldDescription>
```

Comment:
I don't see the nice badge here when my current plan card is selected?

![screenshot-2024-10-30-at-14 59 10](https://github.com/user-attachments/assets/48c00f8a-51f2-481e-a498-ba6ea7769ed7)

Reply:
because you haven't selected another option ?

Reply:
what did you expect ?